client/src/controllers/InitController.ts

```diff
@@ -14,20 +14,41 @@ import { debounce } from '../utils/debounce';
  * <div class="keep-me hide-me" data-controller="w-init" data-w-init-event-value="custom:event other-custom:event">
  *   When the DOM is ready, two additional custom events will be dispatched; `custom:event` and `other-custom:event`.
  * </div>
```

Comment:
Can you also update the opening statement in the JSDoc, it appears this was not updated when we added event dispatching. something like...

```js
/**
 * Adds the ability for a controlled element to dispatch an event and also
 * add or remove classes when ready to be interacted with.
 * 
```